src/index.js

```diff
@@ -28,6 +29,7 @@ const App = () => (
           <Route path="/download-form" component={DownloadForm} />
           <Route path="/download-started" component={DownloadStarted} />
           <Route path="/download-complete" component={DownloadComplete} />
+          <Route path="/download-upload-history" component={DownloadUploadHistoryButton} />
```

Comment:
ideally this links to a `DownloadUploadHistory` component that is `connect`ed to redux, and then you just load `DownloadUploadHistoryButton` with props. take a look at the other components like `DownloadForm`